_episodes/45.md

```diff
@@ -0,0 +1,258 @@
+
+Both Dolphin Scheduler and Airflow are designed to be scalable and highly
+available which supports large-scale distributed environments well.
+
+Airflow supports a wide range of third-party integrations, including popular
+data processing frameworks such as Trino, Spark, and Flink, as well as with
+cloud services such as AWS and GCP. Dolphin Scheduler supports a similar range
+of data processing frameworks and tools making both versatile platforms for
+managing diverse data processing tasks. Airflow is certainly ahead which makes
+it a more flexible solution for some workloads.
+
+Airflow focuses on workflow as code, meaning Airflow uses Python code to create
+and manage workflows. This limits the management and access to these workflows
+to platform engineers and doesn't enable the option for collaborating data
+management across domains. While there are third part tools that make it
+possible to create Airflow DAGs in a GUI, most of them are not up-to-date with
+the latest Airflow versions and not full-featured.
+
+DolphinScheduler project believes that future data governance belongs to a
+data civilization and should not be centralized to a single team.
+Product-focused engineering teams should have access to data and be able
+to orchestrate workflows without the need for extensive coding skills.
+DolphinScheduler uses a drag and drop web UI to create and manages workflows
+while also providing programmatic access using tools like Python SDK and Open
+API.
+
+A positive side effect of DolphinScheduler supporting users outside the data 
+team through a UI is that it offers robust security features. This includes
+authentication, authorization, and data encryption, to ensure that users' data
+and workflows are protected. Airflow has limited built-in security features,
+which may require additional configuration and customization to ensure that
+users' data and workflows are properly protected.
+
+Compared to some other workflow orchestration tools, DolphinScheduler has
+relatively limited documentation and community support since they are a newer
+community, which may make it more challenging for users to get started with the
+platform. However, they have full awareness of this and are investing in
+building out the community including joining talks like this.
+
+## How does DolphinScheduler deal with failures?
+
+Failure is an inevitable aspect of data workflow orchestration. The merits of
+many of these orchestration tools come from 
+
+### Does DolphinScheduler have an alarm mechanism itself?
+
+Apache DolphinScheduler has an alarm mechanism that allows users to receive
+notifications and alerts when certain events or conditions occur within their
+workflows. This mechanism is designed to help users monitor and manage their
+workflows more effectively and respond quickly to any issues or errors that may
+arise.
+
+<img src="/assets/episode/45/alerts.png">
+
+The alarm mechanism in DolphinScheduler is implemented through a set of
```

Comment:
This seems to repeat the earlier stuff .. so we can shorten. The main addition is the email, SMS and that included workflow stuff help